warehouse-management-service/db/migrations/000002_create_shelf_table.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS shelf(
```

Comment:
`shelf` -> `shelves`?

Reply:
Since the table represents all the shelves. Ignore this if you're following some golang conventions for naming

Reply:
^ applies to all the other tables as well

Reply:
This isn't a Go convention. 
I found very divided and opinionated views on using the singular vs plural naming convention.  And I seemed to reason better with singular names (One such [discussion](https://stackoverflow.com/questions/338156/table-naming-dilemma-singular-vs-plural-names) I read)


>Since the table represents all the shelves

However, you mention that since it represents all it is named so, curious if this is a preference or semantically more appropriate

Reply:
Hmm, I didn't realize there was so much debate on this. Using plurals seems  semantically more accurate since the table is a collection of rows, each of which represent an instance of something.  A collection of `item` records referred to collectively as `item` just seems silly to me.

I guess it doesn't matter all that much as long as you're being consistent.